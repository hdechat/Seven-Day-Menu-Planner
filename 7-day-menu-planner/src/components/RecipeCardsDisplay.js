import React from 'react';

const RecipeCardsDisplay = (props) => {
  return (
    <div>
    </div>
  )
}

export default RecipeCardsDisplay;