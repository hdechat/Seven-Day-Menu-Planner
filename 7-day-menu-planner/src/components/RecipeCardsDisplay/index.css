.recipe-cards-display {
  display: flex;
  flex-wrap: wrap;
  justify-content: space-around;
  margin-top: 1rem;
}

.home-display {
  width: 300px;
}

@media screen and (max-width: 350px) {
  .home-display {
    width: 475px;
  }
}