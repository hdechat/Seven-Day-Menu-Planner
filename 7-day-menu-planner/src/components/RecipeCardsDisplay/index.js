import React from 'react';
import RecipeCard from '../RecipeCard'

const RecipeCardsDisplay = ({ recipeCards }) => {
  const cards = recipeCards.map(recipe => {
    return (
      <RecipeCard key={recipe.title} recipe={recipe} />
    )
  });
  return (
    <div>
     {cards}
    </div>
  )
}

export default RecipeCardsDisplay;