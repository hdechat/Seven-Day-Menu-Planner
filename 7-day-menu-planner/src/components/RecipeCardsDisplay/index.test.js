import React from 'react';
import RecipeCardsDisplay from './index.js';
import { shallow } from 'enzyme';

describe('Recipe Cards Display', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<RecipeCardsDisplay />)
  });
  it('matches snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});