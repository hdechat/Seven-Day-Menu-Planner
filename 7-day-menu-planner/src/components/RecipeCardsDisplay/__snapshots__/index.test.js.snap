// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Recipe Cards Display matches snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <RecipeCardsDisplay
    recipeCards={
        Array [
            Object {
              "title": "recipe 1",
            },
            Object {
              "title": "recipe 2",
            },
          ]
    }
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="home-display"
>
          <RecipeCard
                    displayAddToMenu={true}
                    recipe={
                              Object {
                                        "title": "recipe 1",
                                      }
                    }
          />
</div>,
        <div
          className="home-display"
>
          <RecipeCard
                    displayAddToMenu={true}
                    recipe={
                              Object {
                                        "title": "recipe 2",
                                      }
                    }
          />
</div>,
      ],
      "className": "recipe-cards-display",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <RecipeCard
            displayAddToMenu={true}
            recipe={
                        Object {
                                    "title": "recipe 1",
                                  }
            }
/>,
          "className": "home-display",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": "recipe 1",
          "nodeType": "function",
          "props": Object {
            "displayAddToMenu": true,
            "recipe": Object {
              "title": "recipe 1",
            },
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <RecipeCard
            displayAddToMenu={true}
            recipe={
                        Object {
                                    "title": "recipe 2",
                                  }
            }
/>,
          "className": "home-display",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": "recipe 2",
          "nodeType": "function",
          "props": Object {
            "displayAddToMenu": true,
            "recipe": Object {
              "title": "recipe 2",
            },
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        "type": "div",
      },
    ],
    "type": "section",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="home-display"
>
            <RecipeCard
                        displayAddToMenu={true}
                        recipe={
                                    Object {
                                                "title": "recipe 1",
                                              }
                        }
            />
</div>,
          <div
            className="home-display"
>
            <RecipeCard
                        displayAddToMenu={true}
                        recipe={
                                    Object {
                                                "title": "recipe 2",
                                              }
                        }
            />
</div>,
        ],
        "className": "recipe-cards-display",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <RecipeCard
              displayAddToMenu={true}
              recipe={
                            Object {
                                          "title": "recipe 1",
                                        }
              }
/>,
            "className": "home-display",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": "recipe 1",
            "nodeType": "function",
            "props": Object {
              "displayAddToMenu": true,
              "recipe": Object {
                "title": "recipe 1",
              },
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <RecipeCard
              displayAddToMenu={true}
              recipe={
                            Object {
                                          "title": "recipe 2",
                                        }
              }
/>,
            "className": "home-display",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": "recipe 2",
            "nodeType": "function",
            "props": Object {
              "displayAddToMenu": true,
              "recipe": Object {
                "title": "recipe 2",
              },
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "type": "div",
        },
      ],
      "type": "section",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
